R/autosyn.R

```diff
@@ -425,7 +442,7 @@ flywire_ntplot <- function(x, nts=c("gaba", "acetylcholine", "glutamate",
     dopamine = "#CF6F6C"
   )[nts]
 
-  ggplot2::qplot(top.p, fill=top.nt, xlab = 'probability', data=x) +
+  ggplot2::qplot(x$top.p, fill=x$top.nt, xlab = 'probability', data=x) +
```

Comment:
@alexanderbates I think this is ok, but you need to be careful that it doesn't affect the ability to make subsequent modifications by chaining in additional ggplot statements. This is a case where R CMD check may need to be silenced (e.g. by noting a global variable) rather than "fixed" like this.